debugger/src/components/Debugger/Signals/Signal/Action/index.js

```diff
@@ -17,7 +18,7 @@ function getActionName (action) {
 }
 
 function getLineNumber (error) {
-  const variable = error.name === 'TypeError' && error.message.match(/'(.*?)'/) ? error.message.match(/'(.*?)'/)[1] : error.message.split(' ')[0]
+  const variable = error.name === 'TypeError' && String(error.message).match(/'(.*?)'/) ? String(error.message).match(/'(.*?)'/)[1] : String(error.message).split(' ')[0]
```

Comment:
Fixed, also fixed that line does not show when line is 0 (has to do with not always finding the line